### ISSUE TITLE
`scaffold` command adds an unneeded newline at the end of output

### ISSUE THREAD

**Issue by suzuki-shunsuke** [aquaproj/registry-tool#276]
- https://github.com/aquaproj/aqua-registry/pull/9556#issuecomment-1411629843

## How to reproduce

```console
$ aqua-registry -v
aqua-registry version 0.1.6 (291d904ed1c7b8954068c1dada00dfc7353dd7ae)

$ aqua -v
aqua version 1.32.2 (4f892abd43f2781d1252f794072d59a64a369857)
```

```console
$ aqua-registry scaffold soh335/shukujitsu
```

```yaml
packages:
  - type: github_release
    repo_owner: soh335
    repo_name: shukujitsu
    asset: shukujitsu_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
    supported_envs:
      - linux
      - darwin
    checksum:
      type: github_release
      asset: checksums.txt
      file_format: regexp
      algorithm: sha256
      pattern:
        checksum: "^(\\b[A-Fa-f0-9]{64}\\b)"
        file: "^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
# unnecessary newline
```

**Comment by suzuki-shunsuke** [comment]
`aqua gr` doesn't add a newline.

```console
aqua gr soh335/shukujitsu > test.yaml
```

```yaml
packages:
  - type: github_release
    repo_owner: soh335
    repo_name: shukujitsu
    asset: shukujitsu_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
    supported_envs:
      - linux
      - darwin
    checksum:
      type: github_release
      asset: checksums.txt
      file_format: regexp
      algorithm: sha256
      pattern:
        checksum: "^(\\b[A-Fa-f0-9]{64}\\b)"
        file: "^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
```

**Comment by suzuki-shunsuke** [comment]
I found the cause.

https://github.com/aquaproj/registry-tool/blob/e7f7b550dea1fe2bb27122609fd461f53d5c08c7/pkg/scaffold/api.go#L72-L74

https://github.com/aquaproj/registry-tool/blob/e7f7b550dea1fe2bb27122609fd461f53d5c08c7/pkg/patchchecksum/patch.go#L55

- https://github.com/aquaproj/aqua/pull/1541

**Comment by suzuki-shunsuke** [comment]
I think `PatchChecksum` isn't needed anymore, because this is for old aqua.
When we released checksum verification, we expected users use old aqua which didn't support checksum verification, so we patched checksum settings in registry-tool.
registry-tool is managed by aqua, so we can force users to update registry-tool automatically.

The drawback of `PatchChecksum` is that we have to maintain this function.
